gcloud_bigtable/google/bigtable/admin/table/v1/bigtable_table_service_api.py

```diff
@@ -0,0 +1,211 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# EDITING INSTRUCTIONS
+# This file was generated from google/bigtable/admin/table/v1/bigtable_table_service.proto,
+# and updates to that file get reflected here through a regular refresh
+# process. However, manual and updates to that file get reflected here
+# through a regular refresh additions are allowed because the refresh
+# process performs a 3-way merge in order to preserve those manual additions.
+# In order not to break the refresh process, only certain types of
+# modifications are allowed.
+#
+# Allowed modifications:
+# 1. New methods (these should be added to the end of the class)
+# 2. "Notes specific to this wrapper method" sections in the method
+#    documentation
+#
+# Happy editing!
+
+from google.bigtable.admin.table.v1 import bigtable_table_data_pb2
+from google.bigtable.admin.table.v1 import bigtable_table_service_messages_pb2
+from google.bigtable.admin.table.v1 import bigtable_table_service_pb2
+from google.gax import api_callable
+from google.gax import api_utils
+from google.gax import page_descriptor
+
+class BigtableTableServiceApi(object):
+  """
+  Service for creating, configuring, and deleting Cloud Bigtable tables.
+  Provides access to the table schemas only, not the data stored within the tables.
+  """
+
+  # The default address of the logging service.
+  _SERVICE_ADDRESS = "bigtabletableadmin.googleapis.com"
+
+  # The default port of the logging service.
+  _DEFAULT_SERVICE_PORT = 443
+
+  # The scopes needed to make gRPC calls to all of the methods defined in
+  # this service
+  _ALL_SCOPES = [
+      'https://www.googleapis.com/auth/bigtable.admin',
+      'https://www.googleapis.com/auth/bigtable.admin.table',
+      'https://www.googleapis.com/auth/cloud-bigtable.admin',
+      'https://www.googleapis.com/auth/cloud-bigtable.admin.table',
+      'https://www.googleapis.com/auth/cloud-platform'
+  ]
+
+  def __init__(
+      self, service_path=_SERVICE_ADDRESS, port=_DEFAULT_SERVICE_PORT,
+      channel=None, ssl_creds=None, scopes=_ALL_SCOPES,
+      is_idempotent_retrying=True, max_attempts=3, timeout=30):
+
+    self.defaults = api_callable.ApiCallableDefaults(
+      timeout=timeout, max_attempts=max_attempts,
+      is_idempotent_retrying=is_idempotent_retrying)
+
+    self.stub = api_utils.create_stub(
+      bigtable_table_service_pb2.beta_create_BigtableTableService_stub, service_path, port,
+      ssl_creds=ssl_creds, channel=channel, scopes=scopes)
+
+  def __enter__(self):
+    return self
+
+  def __exit__(self, type, value, traceback):
+    self.close()
+
+  def close(self):
+    del self.stub
+
+  # Properties
+  @property
+  def channel(self):
+    return self.channel
+
+  # Page descriptors
+
+  # Service calls
+  def create_table(self, name="", table_id="", table=bigtable_table_data_pb2.Table(), initial_split_keys=[], **kwargs):
```

Comment:
:+1: to @tseaver 

To avoid the pain you could just have your autogenerator use `None` as every default and do a `if foo is None` check at the beginning of the method / function.